### PR TITLE
Update access-procedure.md

### PR DESCRIPTION
**Pull request by ignacio-rocco**
PhD students should select the option "Membre non permanent sauf doctorant, post-doctorant et étudiant en stage de master 2", in order to avoid the error "Votre statut ne permet pas de créer un dossier pour cet appel".

### CODE REVIEW

File: access-procedure.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ is detailed in the sections below.
 
 1. Create an [EDARI account](https://www.edari.fr/user/register) (simple
    personal details). See [this](#edari-account-administrative-account)
-   for all the details about this step.
+   for all the details about this step. WARNING: PhD students should select the option
```
[the remark this answers is not in the document]
Comment (Nemaaar) [reply access-procedure.md:22]: Dear

Thank you for the threads, I have made an account with non-permanent access expect masters, PhD and post-docs but i can not make any access yet, what is the other procedure to make an account for HPC calculations through preparatory access portal?

**Comment by lesteve** [comment]
Coming back from holidays:

> PhD students should select the option "Membre non permanent sauf doctorant, post-doctorant et étudiant en stage de master 2", in order to avoid the error "Votre statut ne permet pas de créer un dossier pour cet appel".

I am not sure I understand. It seems like you are saying that PhD students should not tell they are PhD students because if they tell the truth they won't be able to create their own projects.

If that's the case this is new and this should be reported (of course the right question is to whom and the answer is I don't really know but at least we can try).

**Comment by mdiazmel** [comment]
Hi @lesteve ,

> I am not sure I understand. It seems like you are saying that PhD students should not tell they are PhD students because if they tell the truth they won't be able to create their own projects.
> 
> If that's the case this is new and this should be reported (of course the right question is to whom and the answer is I don't really know but at least we can try).

Yes, that's the case. (I was perplexed to known that, too!). This is the second case I know of. For the first one (a french PhD student) I suggested him to contact the helpdesk. The answer was that this is "normal procedure" if you want to create your own project.

**Comment by mypey** [comment]
Hi @mdiazmel and @lesteve,

there is definitely a problem with the description of the different status on the DARI site. It is really unclear, and maybe the opposite of what it should be. We (IDRIS) told GENCI about it and hope they will clarify the situation soon. We think the list of possibilities should rather be something like that:

1. Membre non permanent (type CDD sauf doctorant, post-doctorant et étudiant en stage de master 2)
2. Membre non permanent (doctorant, post-doctorant et étudiant en stage de master 2)
3. Membre permanent (type CDI)

**Comment by lesteve** [comment]
Thanks @mypey for your insights!

If you ask me, what we want is very simple: a PhD student can create a project, which was the case at the beginning. This feels really unproductive to prevent PhD students to create a project, it just adds unnecessary hurdles and potentially interested people will give up.

For problems like this, I am not even sure who we need to contact, do you know @mypey?

**Comment by mypey** [comment]
What you want should be the reality. For Dynamic Accesses, a PhD student should be able to be in charge of a project ("responsable de  projet"), thus to create a project on the DARI site. This is clear in the GENCI doc (http://www.genci.fr/sites/default/files/Modalitesdacces.pdf, last section page 12): 

"Un  responsable  scientifique  est  désigné  pour  tout  dossier  de  demande  d’heures  de  calcul. Ce responsable peut être un membre  permanent  du  laboratoire  d’appartenance ou  un  étudiant stagiaire en master 2, doctorant ou post-doctorant, qu’il soit académique ou industriel."

This particular problem should be fixed once the person in charge comes back from vacation (next week as far as I know). Until then, you can keep selecting "Membre non permanent sauf doctorant, post-doctorant et étudiant en stage de master 2".

For other problems about hours allocation via the DARI portal, you can contact GENCI. As you are researchers, acces@genci.fr seems to be the right contact (http://www.genci.fr/fr/content/page-contact).

**Comment by RemiLacroix-IDRIS** [comment]
> If you ask me, what we want is very simple: a PhD student can create a project, which was the case at the beginning.

PhD students are still allowed to create an "Accès Dynamique" project. This is solely an user interface issue.

**Comment by lesteve** [comment]
This should have been fixed, according to IDRIS support people. Closing, feel free to reopen if that's not the case.

**Pull request closed without merging**